.gitignore

```diff
@@ -8,3 +8,5 @@ docs/book/
 *.db
 *.db*
 
+
+.idea
```

Comment:
typically @adlerjohn has wanted us to avoid ide specific settings in the shared .gitignore, and for ppl to update their local `.git/info/exclude` instead.